include/EVT/io/CANopen.hpp

```diff
@@ -6,7 +6,7 @@
 #ifndef _EVT_CANopen_
 #define _EVT_CANopen_
 
-#include <Canopen/co_if.h>
+#include "co_core.h"
```

Comment:
I know there was something of a fiasco with the CANopen includes. Can we still use `<>` here instead of `""`?
```suggestion
#include <co_core.h>
```